audit-connector/src/test/java/com/sequenceiq/cloudbreak/audit/converter/AttemptAuditEventResultToGrpcAttemptAuditEventResultConverterTest.java

```diff
@@ -49,7 +63,18 @@ void testPreventPossibleNullValuesServiceApiRequestResult() {
                 .withResourceCrns(CRNS)
                 .build();
         AttemptAuditEventResult source = makeMinimalAttemptAuditEventResult(resultServiceEventData);
+        underTest = new AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter(createMockUtilizer(ResultServiceEventData.class));
+
         underTest.convert(source);
+        verify(mockAuditEventDataUtilizer, times(1)).modifyAuditEventBuilderWithEventData(any(), any());
+    }
+
+    @Test
+    void testWhenResultEventDataIsNullThenNoUtilizerCallHappens() {
+        AttemptAuditEventResult source = makeMinimalAttemptAuditEventResult(null);
+
+        underTest.convert(source);
+        verify(mockAuditEventDataUtilizer, times(0)).modifyAuditEventBuilderWithEventData(any(), any());
```

Comment:
times(0) -> never() (although its just cosmetics)

Reply:
oh, this `never()` stuff is new to me, thanks!